cli/tools/fmt.rs

```diff
@@ -63,8 +61,38 @@ pub async fn format(
   Ok(())
 }
 
+/// Formats markdown (using https://github.com/dprint/dprint-plugin-markdown) and its code blocks
+/// (ts/tsx, js/jsx).
+fn format_markdown(
+  file_text: &str,
+  ts_config: dprint_plugin_typescript::configuration::Configuration,
+) -> Result<String, String> {
+  let md_config = get_markdown_config();
+  dprint_plugin_markdown::format_text(
+    &file_text,
+    &md_config,
+    Box::new(move |tag, text, line_width| {
+      let tag = tag.to_lowercase();
+      if matches!(
+        tag.as_str(),
+        "ts" | "tsx" | "js" | "jsx" | "javascript" | "typescript"
+      ) {
+        let mut codeblock_config = ts_config.clone();
+        codeblock_config.line_width = line_width;
+        dprint_plugin_typescript::format_text(
+          &PathBuf::from("_rand.".to_owned() + &tag),
```

Comment:
I'm not sure what this will do when the tag is "javascript" or "typescript", but I think this needs to be updated.